src/components/ui/timeline.js

```diff
@@ -72,15 +72,17 @@ function Timeline({ items, transactions }) {
               pl={listPadding}
               as="li"
               pos="relative"
-              _before={{
-                content: `' '`,
-                position: 'absolute',
-                left: 'calc(0.75rem / 2)',
-                width: '1px',
-                height: index === items.length - 1 ? '16px' : '200%',
-                bg: 'gray.300',
-                top: index === 0 && '16px',
-              }}
+              _before={
+                range?.subItems?.length > 1 && {
```

Comment:
`range.subItems`, bir date'in içindeki işlem sayısını gösteriyor. bakmamız gereken şeyin bu olduğundan emin değilim 😥